plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/ElasticsearchQueryRunner.java

```diff
@@ -53,11 +52,24 @@ public static DistributedQueryRunner createElasticsearchQueryRunner(
             Map<String, String> extraConnectorProperties,
             int nodeCount)
             throws Exception
+    {
+        return createElasticsearchQueryRunner(address, tables, extraProperties, extraConnectorProperties, nodeCount, "elasticsearch");
+    }
+
+    public static DistributedQueryRunner createElasticsearchQueryRunner(
+            HostAndPort address,
+            Iterable<TpchTable<?>> tables,
+            Map<String, String> extraProperties,
+            Map<String, String> extraConnectorProperties,
+            int nodeCount,
+            String catalogName)
+            throws Exception
     {
         RestHighLevelClient client = null;
         DistributedQueryRunner queryRunner = null;
         try {
-            queryRunner = DistributedQueryRunner.builder(createSession())
+            queryRunner = DistributedQueryRunner.builder(testSessionBuilder()
+                            .setCatalog(catalogName).setSchema(TPCH_SCHEMA).build())
```

Comment:
nit: pre-existing convention in the codebase (makes reading long chains easier)

```suggestion
                            .setCatalog(catalogName)
                            .setSchema(TPCH_SCHEMA)
                            .build())
```